src/app/services/technical-record-http/technical-record-http.service.ts

```diff
@@ -38,97 +40,76 @@ export class TechnicalRecordHttpService {
-      primaryVrm: recordCopy.vrms ? recordCopy.vrms[0].vrm : null,
-      trailerId: recordCopy.trailerId ?? null,
-      techRecord: recordCopy.techRecord
-    };
+  getRecordV3(systemNumber: string, createdTimestamp: string): Observable<V3TechRecordModel[]> {
+    const url = `${environment.VTM_API_URI}/v3/technical-records/${systemNumber}/${createdTimestamp}`;
 
-    return this.http.post<PostNewVehicleModel>(`${environment.VTM_API_URI}/vehicles`, body);
+    return this.http.get<V3TechRecordModel[]>(url, { responseType: 'json' });
   }
 
-  createProvisionalTechRecord(
-    systemNumber: string,
-    techRecord: TechRecordModel,
-    user: { id?: string; name: string }
-  ): Observable<VehicleTechRecordModel> {
-    // THIS ALLOWS US TO CREATE PROVISIONAL FROM THE CURRENT TECH RECORD
-    const recordCopy = cloneDeep(techRecord);
-    recordCopy.statusCode = StatusCodes.PROVISIONAL;
-    delete recordCopy.updateType;
-
-    const url = `${environment.VTM_API_URI}/vehicles/add-provisional/${systemNumber}`;
+  createVehicleRecord(newVehicleRecord: V3TechRecordModel): Observable<V3TechRecordModel> {
+    const recordCopy: TechRecordPUT = cloneDeep(newVehicleRecord);
 
     const body = {
-      msUserDetails: { msOid: user.id, msUser: user.name },
-      techRecord: [recordCopy]
+      ...recordCopy
     };
 
-    return this.http.post<VehicleTechRecordModel>(url, body, { responseType: 'json' });
+    return this.http.post<V3TechRecordModel>(`${environment.VTM_API_URI}/v3/technical-records`, body);
   }
+  //V3 are we still in need of this?
+  // createProvisionalTechRecord(techRecord: V3TechRecordModel): Observable<V3TechRecordModel> {
+  //   // THIS ALLOWS US TO CREATE PROVISIONAL FROM THE CURRENT TECH RECORD
+  //   const recordCopy = cloneDeep(techRecord);
+  //   recordCopy.techRecord_statusCode = StatusCodes.PROVISIONAL;
+  //   delete recordCopy.techRecord_updateType;
 
-  updateTechRecords(
-    systemNumber: string,
-    vehicleTechRecord: VehicleTechRecordModel,
-    user: { id?: string; name: string },
-    recordToArchiveStatus?: StatusCodes,
-    newStatus?: StatusCodes
-  ): Observable<PutVehicleTechRecordModel> {
-    const newVehicleTechRecord = cloneDeep(vehicleTechRecord);
+  //   const url = `${environment.VTM_API_URI}/vehicles/add-provisional/${techRecord.systemNumber}`;
 
-    const newTechRecord = newVehicleTechRecord.techRecord[0];
+  //   const body = {
+  //     techRecord: [recordCopy]
+  //   };
 
-    newTechRecord.statusCode = newStatus ?? newTechRecord.statusCode;
-    delete newTechRecord.updateType;
+  //   return this.http.patch<V3TechRecordModel>(url, body, { responseType: 'json' });
+  // }
 
-    const url = `${environment.VTM_API_URI}/vehicles/${systemNumber}` + `${recordToArchiveStatus ? '?oldStatusCode=' + recordToArchiveStatus : ''}`;
+  updateTechRecords(techRecord: V3TechRecordModel): Observable<V3TechRecordModel> {
+    console.log(techRecord);
+    const body = { ...techRecord };
```

Comment:
Don't technically need this anymore if we're not actually doing anything further with the tech record before sending it out